Src/Support/CommonProjectProperties.xml

```diff
@@ -3,8 +3,8 @@
   <!-- common nupkg information -->
   <PropertyGroup>
     <Version>1.51.0</Version>
-    <Authors>Google Inc.</Authors>
-    <Copyright>Copyright 2017 Google Inc.</Copyright>
+    <Authors>Google LLC</Authors>
+    <Copyright>Copyright 2021 Google LLC</Copyright>
```

Comment:
Support is handwritten, so it might make more sense to leave it on 2017. But I'm fine eitherway.

Reply:
We seem to have been updating pretty much everything - and certainly they existed before 2017 - so I thought I might as well.